viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -333,6 +333,14 @@ export class Cognite3DViewer {
     });
   }
 
+  /**
+   * Set resolution threshold.
+   * @param threshold
+   */
+  setResolutionThreshold(threshold: number): void {
```

Comment:
This means nothing to me. Is this what should be named `setMaximumRenderResolution`? Also, document how it works (i.e. that its width x height) - and what happens when resolution cap is exceeded.